model/src/main/kotlin/OrtResult.kt

```diff
@@ -326,12 +311,21 @@ data class OrtResult(
      * Return true if the project or package with the given identifier is excluded.
      */
     @Suppress("UNUSED") // This is intended to be mostly used via scripting.
-    fun isExcluded(id: Identifier) =
-        analyzer?.result?.let { repository.config.excludes?.isExcluded(id, this) } == true
+    fun isExcluded(id: Identifier): Boolean
+        = getExcludes().isExcluded(id, this)
 
     /**
      * Return a copy of this [OrtResult] with the [Repository.config] replaced by [config].
      */
     fun replaceConfig(config: RepositoryConfiguration): OrtResult =
         copy(repository = repository.copy(config = config)).also { it.data += data }
+
+    private fun getPackages(): Set<CuratedPackage>
+            = analyzer?.result?.packages ?: emptySet()
+
+    private fun getProjects(): Set<Project>
+            = analyzer?.result?.projects ?: emptySet()
+
+    private fun getExcludes(): Excludes
+            = repository.config.excludes ?: Excludes()
```

Comment:
These three functions would fit in a single line each.

Reply:
Done.